documentation/modules/auxiliary/admin/http/gitlab_password_reset_account_takeover.md

```diff
@@ -0,0 +1,71 @@
z71UtTMGw2NldTIGmvMmnOz/5klDQ3s3UMpMVfsW1WdV2oyd3dkKEk+8pS/u0lG8at9EbVnf+qGROrKmAxXqJnF8nBFXvbRoRTl3TSgvDu00uNbfxbPk9qnTh7o7voYaJ7ptSuHejTmQMrZabuktjxl9Hnj2XMs5SxRIz41h6GJ3O3KYxM/+WSiWHS1d9LqoMXWvYpkycBE9qV0vV01Ey2kG0oO9sm8HhjP2o2zs63zWnaWnJtGZffUrmvJ2jNvE1MOdefVzY2zZ+/OSAmodp3UuTlClaPFnuFDY93O1Gqo6/U6lFJVu5uw24lvo6SG7A7rsfFRu6YhySHTVSm44zdtKd3xoeTmTGpWek22Uo1mN89Sh0hy8Cyy9H6UZE8a3hNZhoK4iw6nKO+mAeXdYZaf3TobrDzELXVl1FPeX57lNPKodO1m5dSp56LlA8MHj6RLtcpRkIz1qWbFX+qdKOeQfap8yf45zzJRk2EN7miNgzKph6KVDdRCEtmXarfZ56y6SvG92Mzb7HPtxlmSpxJL17Jz80tDltc2CwJLbnKWMfPaRwHt0Iz1dMlKy0yERtn/7KeTkZpso95bJvWObfZHjWTiOJ7XN+Lo5i9GqfuW9dZ5JQ+sdlt6962RfXxVc2U6qXbHhJKz0k5iycorUm7uSkN76y2yvc6iCDZKNZTGLq2nb1SC56bUHUaHTZR304DyDk1dOm+uvvrqZQeh09UthqYiFVipsRBaH1HeTQPKOzR16fzOVzgINaK8Q2h3RXk3DSjv0NT13ve+t/dnWwhJlHcI7a4o76YB5R1CCCGERorybhpQ3iGEEEJopCjvpsGelnfNXAAAALAPLI/hPRDl3TSgvAMAAJgZy2N4D0R5Nw0o7wAAAGbG8hjeA1HeTQPKOwAAgJmxPIb3QJR304DyDgAAYGYsj+E9EOXdNKC8AzgofvT/vP5njvxlw4vf9DuN267wvRf8wv/0zO9qjBNF+/ZTb/v92SwHYC9YHsN7IMq7aUB5B3BQdMuUZ373D738uj/c9QpvZvUQ5R3AliyP4T0Q5d00oLwDOCh6y5Tv/8nXqcJTnVeNZwjlHcBhY3kM74Eo76YB5R3AQdFbpshYy7sXv+l3/EPbn37PZ777Ry+0UUM0sGvv9VfAXk9fUpyXXHWHHWoy8pS/7SJfKPr7RRtrnplFqG1LAjph20UtYXsXKE9l5TUmmt008PzXf6imCgBdlsfwHojybhpQ3gEcFLUAMi6eXNC4JEpdpZIoBZCMsSfICv/uRMY1k4fU4U5DEeymhtz0ah+nJ5KGLqVc04wvv+5Tek3DRiWT3DzQ0ZKYwtaEPaP91cglJ9O7HAAIy2N4D0R5Nw0o7wAOChcxDSmeVM289JoH8vWYqFVRSqUw5K+GYvbWQ7KnLBOKkIKskmrPjWQYmjjGedpZr/7KTT6J1kzX+CdgtRstqnc5ABCWx/AeiPJuGlDeARwUKllSprjoqUWbv6ZqsEMu5Tut1f51okpjT+HlruuqhLJdQ9yt9ZwH2l7rMLWdgF5/6KJfUTQFUcIuQ9WuQap/TazJqrkKAL0sj+E9EOXdNKC8AzgomjJFdU/9CaauNtVPQ8ovF3kr/IfqocaeQsoNRXYy3QIrDs2MCliLPBdw3/0jL9MscpPROGy3vJOd8g5gV1gew3sgyrtpQHkHcFB0yxRZVBu5lNFr/XJuCA1XEA1c4T9UD8leCyyN9U9Lm8JLFkWuBZbpFl4m09khRZviyP7Sax6Qg7srfjibhKvdKFquAkAvy2N4D0R5Nw0o7wAOilrEGJcyNta2LrlU0hDbXTCJFElD/mo35VrQ1XxFV8PWStFxXHTWmHZzWFkS33Hio4D5Ms/DE7lJWD65pHbsouajtgLWqwDQZXkM74Eo76YB5R3AQdEUMcblSy251DWpmVJyGQ2xfUv/eBonkF+MkpLRl2wU/mtzvqoaS5WW7Sm5hMs4U+PUysxuzZIzsLp1d8bbIrQQfjEKwJYsj+E9EOXdNKC8Azi0dKsoAJgHy2N4D0R5Nw0o7wAOLZR3AHNleQzvgSjvpgHlHcChhfIOYK4sj+E9EOXdNKC8AwAAmBnLY3gPRHk3DSjvAAAAZsbyGN4DUd5Ngz0t7xBCCCE0J1HeTQPKO4QQQgiNFOXdNKC8QwghhNBIUd5NA8o7hBBCCI0U5d00oLxDCCGE0EhR3k0DyjuEEEIIjRTl3TSgvEMIIYTQSFHeTQPKO4QQQgiNFOXdNKC8QwghhNBIUd5NA8o7hBBCCI0U5d00oLxDCCGE0EhR3k0DyjuEEEIIjRTl3TSgvEMIIYTQSFHeTQPKO4QQQgiNFOXdNKC8QwghhNBIUd5NA8o7hBBCCI0U5d00oLxDCCGE0EhR3k0DyjuEEEIIjRTl3TSgvEMIIYTQSFHeTQPKO4QQQgiNFOXdNKC8QwghhNBIUd5NA8o7hBBCCI0U5d00oLxDCCGE0EhR3k0DyjuEEEIIjRTl3TSgvEMIIYTQSFHeTQPKO4QQQgiNFOXdNKC8QwghhNBIUd5NA8o7hBBCCI0U5d00oLxDCCGE0EhR3k0DyjuEEEIIjRTl3TSgvEMIIYTQSFHeTYOTJ08+61nPWt40hBBCCKEBqWBQ2dAUEpPjUJR31177jne/+91UeAghhBBaIZUKKhhUNjSFxOQ4FOWd0K1SMf5FhBBCCKEBqVSYQW0nDkt5BwAAAHBIoLwDAAAAmBWUdwAAAACzgvIOAAAAYFZQ3gEAAADMCso7AAAAgFlBeQcAAAAwKyjvAAAAAGYF5R0AAADArKC8AwAAAJgVlHcAAAAAs4LyDgAAAGBWUN4BAAAAzArKOwAAAIBZQXkHAAAAMCso7wAAAABmBeUdAAAAwKygvAMAAACYFZR3AAAAALOC8g4AAABgVlDeAQAAAMwKyjsAAACAWUF5BwAAADArKO8AAAAAZgXlHQAAAMCsoLwDAAAAmBFPP/f/BzUaN4l9xPHuAAAAAElFTkSuQmCC)
```

Comment:
To the tester, test this with the `info -d` command that will open the browser and verify if it renders or not.

Reply:
I confirmed, it renders correctly:
![Screenshot 2024-03-07 at 14 22 34](https://github.com/rapid7/metasploit-framework/assets/56716719/90719cef-2954-4c0f-b4a9-ba8e83fcf9c0)